src/base/static/components/place-detail/survey-response-editor.js

```diff
@@ -133,7 +134,15 @@ class SurveyResponseEditor extends Component {
               submitter={this.props.submitter}
               anonymousName={this.props.placeConfig.anonymous_name}
             />
-            <ActionTime time={this.props.attributes.get("updated_datetime")} />
+            {this.props.appConfig.show_timestamps !== false && (
```

Comment:
Why can't this be `this.props.appConfig.show_timestamps &&` ?

Reply:
We should set the default values in our `config-loader`, instead of scattering implicit default definitions throughout the codebase.

Reply:
https://github.com/jalMogo/mgmt/issues/185

Reply:
Maybe I'm missing something, but it seems like we can set default values by adding a line like this in our `config-loader`:

```javascript
  // set default value for `show_timestamps`:
  if (!Object.hasOwnProperty(config.app.show_timestamps)) {
    config.app.show_timestamps = true;
  }

```

https://github.com/mapseed/platform/blob/master/scripts/config-loader.js#L30

Reply:
Yes, I think that should work. The way the static build process is set up at the moment we'll also need to replicate the same logic in `static-build.js`. Hopefully we can simplify that soon, but in the meantime I think consolidating all the implicit defaults is still a useful thing to do.

Reply:
Ah, I knew I was missing something :grimacing: 

Yes, I would suggest breaking out a separate module like `setConfigDefaults` which exports a function that accepts the config object, or the path to the config file, and returns the config object with the default values. But again, I might be missing something, ha.